src/Draggable/Sensors/MouseSensor/MouseSensor.js

```diff
@@ -74,45 +64,67 @@ export default class MouseSensor extends Sensor {
       return;
     }
 
+    this.startEvent = event;
+
     document.addEventListener('mouseup', this[onMouseUp]);
+    document.addEventListener('mousemove', this[onDistanceChange]);
 
-    const target = document.elementFromPoint(event.clientX, event.clientY);
-    const container = closest(target, this.containers);
+    const container = closest(event.target, this.containers);
 
     if (!container) {
       return;
     }
 
     document.addEventListener('dragstart', preventNativeDragStart);
 
-    this.mouseDown = true;
-
-    clearTimeout(this.mouseDownTimeout);
+    this.currentContainer = container;
     this.mouseDownTimeout = setTimeout(() => {
-      if (!this.mouseDown) {
-        return;
-      }
-
-      const dragStartEvent = new DragStartSensorEvent({
-        clientX: event.clientX,
-        clientY: event.clientY,
-        target,
-        container,
-        originalEvent: event,
-      });
-
-      this.trigger(container, dragStartEvent);
-
-      this.currentContainer = container;
-      this.dragging = !dragStartEvent.canceled();
-
-      if (this.dragging) {
-        document.addEventListener('contextmenu', this[onContextMenuWhileDragging]);
-        document.addEventListener('mousemove', this[onMouseMove]);
-      }
+      this.delayOver = true;
+      if (this.distance < this.options.distance) return;
```

Comment:
I'm surprised the linting isn't complaining for this - I believe we have a preference for "no single line if blocks"

Reply:
I couldn't find it in the `.eslintrc.js`, I'm not even sure there is an option for it since I couldn't find it here: https://eslint.org/docs/rules/

Regardless, I can update it if necessary